TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedDocumentNodes.html

```diff
@@ -0,0 +1,46 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="EditImage"/>
+
+<f:section name="Title">RelatedDocumentNodes view of AssetController</f:section>
+
+<f:section name="Content">
+      <legend>{asset.resource.filename}</legend>
```

Comment:
would probably be better to use the title if available and the label as fallback, like it's done in the overviews